rrule/build.boot

```diff
@@ -4,7 +4,8 @@
 
 (require '[cljsjs.boot-cljsjs.packaging :refer :all])
 
-(def +lib-version+ "2.1.0")
+(def +commit+ "838226d")
+(def +lib-version+ (str "2.1.0-" +commit+))
```

Comment:
The build automatic is harded to read only +lib-version+ and +version+ using just regex so it can't evaluate this.

This should work:
```clj
(def +lib-version+ "2.1.0-838226d")
(def +version+ (str +lib-version+ "-0"))
(def +commit+ (second (re-matches #".*-(.*)" +lib-version+)))
```